frontend/src/pages/Mesh/MeshPage.tsx

```diff
@@ -82,8 +82,9 @@ export type MeshData = {
 };
 
 type MeshPageState = {
+  controller?: Controller;
   meshData: MeshData;
-  lastResizeTime: TimeInMilliseconds; // just a way to force a top-down re-render on a mesh-level resize (e..f targetPanelCollapse)
+  setSelectedIds?: (values: string[]) => void;
```

Comment:
Why do you add `setSelectedIds` in the MeshPageState? If it is just for cypress tests I think we should add a comment indicating it (bit weird to see this kind of methods in a component state)

Reply:
yes, and it is weird.  and yes, I will add something here, I only got this in place last night and then stopped for the day...